tox.ini

```diff
@@ -37,8 +37,7 @@ import-order-style = cryptography
 [testenv:flake8]
 basepython = python2.7
 deps =
-    flake8 <= 2.4.1
-    git+https://github.com/public/flake8-import-order@2ac7052a4e02b4a8a0125a106d87465a3b9fd688
```

Comment:
yes

Back when i added the environment with the flake8 check there was a bug with one of the rules which was fixed in the repo, but not in the release (something with import ordering if i remember  right).